### PR TITLE
docs: add mongoose as a curated topic

### PR DESCRIPTION
**Pull request by ChristianMurphy**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [x] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:
Mongoose is a widely used JavaScript ODBM and has over 2400 repositories the topic at the moment.

/cc @vkarpov15

### CODE REVIEW

File: topics/mongoose/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+created_by: LearnBoost
+display_name: Mongoose
+github_url: https://github.com/Automattic/mongoose
+logo: mongoose.png
+released: 2010
```
Comment (cheshire137) [inline topics/mongoose/index.md]: You could add a `related: mongodb, nodejs, orm` line.

File: topics/mongoose/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+short_description: Mongoose is a MongoDB object modeling tool.
+topic: mongoose
+url: http://mongoosejs.com
+---
+Mongoose is a MongoDB object modeling tool designed to work in an asynchronous environment.
```
Comment (cheshire137) [inline topics/mongoose/index.md]: I think this line could also be the `short_description`. You could then add more details to this description, maybe:

> Mongoose includes built-in type casting, validation, query building, and business logic hooks.

Comment (ChristianMurphy) [reply topics/mongoose/index.md]: added

File: topics/mongoose/index.md (hunk 1 above)
Comment (ChristianMurphy) [reply topics/mongoose/index.md]: added

**Comment by vkarpov15** [comment]
Thanks! Let me know if you need the logo in a particular resolution, I'm happy to provide.

**Comment by ChristianMurphy** [comment]
A [288x288 pixel resolution png](https://github.com/github/explore/blob/master/docs/API.md#logo) copy of the logo would fit well.
Thanks @vkarpov15!

**Comment by vkarpov15** [comment]
image

Let me know if that works or if I need to make any changes :+1:

**Pull request merged**